apilos_settings/services/services_view.py

```diff
@@ -306,15 +330,18 @@ def edit_bailleur(request, bailleur_uuid):
 
 
 @require_GET
-def user_list(request):
+def user_list(request: HttpRequest) -> dict[str, Any]:
     user_list_service = UserListService(
         search_input=request.GET.get("search_input", ""),
         order_by=request.GET.get("order_by", "username"),
         page=request.GET.get("page", 1),
-        my_user_list=request.user.user_list(),
+        my_user_list=(
+            User.objects.exclude(is_staff=True).exclude(is_superuser=True).all()
```

Comment:
Pas sûr que le .all() soit utile, et on peut peut-être faire un OR en db: 

```suggestion
            User.objects.exclude(Q(is_staff=True) | Q(is_superuser=True))
```

Reply:
Ah en fait le plus simple reste peut-être 
`User.objects.filter(is_staff=False, is_superuser=False)`, maintenant que j'y pense 😅

Reply:
Sans importance je suppose